addons/stock_account/models/stock_quant.py

```diff
@@ -88,14 +88,6 @@ def _apply_inventory(self):
             else:
                 super(StockQuant, inventories)._apply_inventory()
 
-    def _get_inventory_move_values(self, qty, location_id, location_dest_id, package_id=False, package_dest_id=False):
-        res_move = super()._get_inventory_move_values(qty, location_id, location_dest_id, package_id, package_dest_id)
-        if not self.env.context.get('inventory_name'):
-            force_period_date = self.env.context.get('force_period_date', False)
-            if force_period_date:
-                res_move['name'] += _(' [Accounted on %s]', force_period_date)
-        return res_move
-
```

Comment:
Instead of deleting this method, replace `res_move['name']` with:

res['inventory_name'] = res.get('inventory_name','') += _(' [Accounted on %s]', force_period_date)